plugins/caddy/go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/darkweak/souin/plugins/caddy
+module github.com/caddyserver/cache-handler
```

Comment:
What's happening here? Are you no longer intending to have a separate Souin caddy module for advanced use, only supporting the `caddyserver/cache-handler` module?

Reply:
Reverted 2 days ago ;)